website/content/en/docs/golang/olm-integration/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Integrating with Operator Lifecycle Manager
+linkTitle: Integrating with OLM
+weight: 200
+---
```

Comment:
I *think* this section should be moved to the top level. I think this stuff isnt go-operator specific?